src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -286,12 +295,30 @@ private BigQueryWriter getWriter() {
     long retryWait = config.getLong(config.BIGQUERY_RETRY_WAIT_CONFIG);
     BigQuery bigQuery = getBigQuery();
     if (updateSchemas) {
-      return new AdaptiveBigQueryWriter(bigQuery, getSchemaManager(bigQuery), retry, retryWait);
+      return new AdaptiveBigQueryWriter(bigQuery, getSchemaManager(bigQuery), retry, retryWait,  metrics);
     } else {
-      return new SimpleBigQueryWriter(bigQuery, retry, retryWait);
+      return new SimpleBigQueryWriter(bigQuery, retry, retryWait, metrics);
     }
   }
 
+  private void configureMetrics() {
+    // todo more metric configuration?
+    metrics = new Metrics();
+    rowsRead = metrics.sensor("rows-read");
+    rowsRead.add(metrics.metricName("rows-written-avg",
+                                    "kafka", // I have no idea how the group name is used or what it should be...
```

Comment:
You really want the source class to be the group name of the metric?  That isn't what seems to be happening in kafka proper.  I was actually going to rename the group as "kcbq" or something.